Cards/card.cpp

```diff
@@ -1118,14 +1118,16 @@ void heal(Card *card, Field &, Field &)
         card->power = card->powerBase;
 }
 
-void heal(Card *card, const int x, Field &ally, Field &enemy)
+void heal(Card *card, const int x, Field &, Field &)
 {
     assert(!card->isSpecial);
 
-    if (card->power < card->powerBase)
-    {
-        const int possibleX = std::min(x, card->powerBase - card->power);
-        card->power += possibleX;
+    if (card->power < card->powerBase) {
+        const int boost = std::min(x, card->powerBase - card->power);
+        card->power += boost;
+        // TODO: check if heal is actually a boost
+        // in case of triggers and other stuff.
+        // probably, its correct to call `boost` here instead
```

Comment:
We should probably find instances. Are there some cards having "whenever smb boost" texts? I dare to guess, not in Monster faction